vo_models/uws/models.py

```diff
@@ -167,39 +177,52 @@ class Jobs(BaseXmlModel, tag="jobs", ns="uws", nsmap=NSMAP):
-                                    /{jobs}/{job-id}/parameters
-    results (Results):              The results for the job - can also be retrieved at /{jobs}/{job-id}/results
-    error_summary (ErrorSummary):   A short summary of an error
-    job_info (Any):                 This is arbitrary information that can be added to the job description by the UWS
-                                    implementation.
-
-    Attributes:
-    version: (UWSVersion)   Note that this attribute is actually required by the 1.1 specification - however remains
-                            optional in the schema for backwards compatibility.
-                            It will be formally required in the next major revision.
+    Parameters:
+        job_id (JobIdentifier, str):
+            (element) - The identifier for the job.
+        run_id (str):
+            (element) - This is a client supplied identifier - the UWS system does nothing other than to return it
+            as part of the description of the job
+        owner_id (str):
+            (element) - The owner (creator) of the job - this should be expressed as a string that can be
+            parsed in accordance with IVOA security standards.
+
+                        If there was no authenticated job creator then this should be set to NULL.
+        phase (ExecutionPhase):
+            (element) - The execution phase.
+        quote (UTCTimestamp):
+            (element) - A Quote predicts when the job is likely to complete.
+        creation_time (UTCTimestamp):
+            (element) - The instant at which the job was created.
+
+                        Note that the version 1.1 of the specification requires that this element
+                        be present. It is optional only in versions 1.x of the schema for backwards compatibility.
+                        2.0+ versions of the schema will make this formally mandatory in an XML sense.
+        start_time (UTCTimestamp):
+            (element) - The instant at which the job started execution.
+        end_time (UTCTimestamp):
+            (element) - The instant at which the job finished execution.
+        execution_duration (int):
+            (element) - The duration (in seconds) for which the job should be allowed to run.
+
+                        A value of 0 is intended to mean unlimited.
+        destruction (UTCTimestamp):
+            (element) - The time at which the whole job + records + results will be destroyed.
+        parameters (Parameters):
+            (element) - The parameters to the job (where appropriate)
+        results (Results):
+            (element) - The results for the job
+        error_summary (ErrorSummary):
+            (element) - A short summary of an error
+        job_info (list[str]):
+            (element) - This is arbitrary information that can be added to the job description by the UWS
+            implementation.
+
+        version: (UWSVersion)
```

Comment:
This is somehow being rendered as (UWSVersion | None). I'm not sure why or how.

![image](https://github.com/spacetelescope/vo-models/assets/653031/11e31234-67c3-41a4-8213-96f3018bb0bd)